src/containers/homepage/survey-list.component.tsx

```diff
@@ -62,6 +84,64 @@ export class SurveyListComponent extends Component<SurveyListComponentProps> {
                 ),
             },
         ];
-        return <Table dataSource={items} columns={columns} bordered />;
+
+        return (
+            <>
+                <Row type="flex" justify="space-between">
+                    <Col span={12}>
+                        <h2>Surveys</h2>
+                    </Col>
+                    <Col span={12}>
+                        <Link to="/survey/new">
+                            <Tooltip placement="left" title="Create a new survey">
+                                <Button type="primary" style={{ float: 'right' }}>
+                                    <Icon type="plus" />
+                                    New survey
+                                </Button>
+                            </Tooltip>
+                        </Link>
+                    </Col>
+                </Row>
+                <Table
+                    dataSource={this.filter(items)}
+                    columns={columns}
+                    bordered
+                    title={() => (
+                        <Row type="flex" justify="space-between">
+                            <Col span={16}>
+                                <span>Filter: &nbsp;</span>
```

Comment:
Why the space followed by a non-breaking space?

Reply:
It prevents consecutive whitespace characters from collapsing into a single whitespace. I will remove it and use css instead.